### ISSUE TITLE
PHP 7 Compatibility

### ISSUE THREAD

**Issue by Okeanos** [piwik/piwik-package#48]
Given that Piwik is compatible with PHP 7.0 for the most part as per [this report](https://github.com/piwik/piwik/issues/8689) and that using the [dotdeb](https://www.dotdeb.org) packages one can install PHP 7.0 on Debian Jessie cleanly, I wonder when/if updating this package's dependencies is planned?
Right now using this package would require installation of PHP 5.x even though PHP 7.x might be available/already installed.

**Comment by aureq** [comment]
Hi @Okeanos 

Thank you for the report. Do you mind to run the following command `dpkg -l | grep php`.
I would like to make sure we capture all the correct requirements.

Please note that while Piwik is almost ready for PHP7, the issue you referenced isn't closed yet. Equally important, not all piwik extensions/plugins may be compatible with PHP7.

**Comment by Okeanos** [comment]
@aureq 

Running `dpkg -l | grep php` yields:

```
ii  php-common                     21-1~dotdeb+8.1              all          Common files for PHP packages
ii  php-readline                   21-1~dotdeb+8.1              all          readline module for PHP [default]
ii  php7.0                         7.0.7-1~dotdeb+8.1           all          server-side, HTML-embedded scripting language (metapackage)
ii  php7.0-apcu                    5.1.4-1~dotdeb+8.1           amd64        APC User Cache for PHP
ii  php7.0-cli                     7.0.7-1~dotdeb+8.1           amd64        command-line interpreter for the PHP scripting language
ii  php7.0-common                  7.0.7-1~dotdeb+8.1           all          Common files for packages built from the PHP source
ii  php7.0-curl                    7.0.7-1~dotdeb+8.1           amd64        CURL module for PHP
ii  php7.0-fpm                     7.0.7-1~dotdeb+8.1           amd64        server-side, HTML-embedded scripting language (FPM-CGI binary)
ii  php7.0-gd                      7.0.7-1~dotdeb+8.1           amd64        GD module for PHP
ii  php7.0-geoip                   1.1.0-4~dotdeb+8.2           amd64        GeoIP module for PHP
ii  php7.0-igbinary                1.2.1-1~2b7c703-1~dotdeb+8.1 amd64        igbinary serializer for PHP
ii  php7.0-imagick                 3.4.2-1~dotdeb+8.1           amd64        Provides a wrapper to the ImageMagick library
ii  php7.0-intl                    7.0.7-1~dotdeb+8.1           amd64        Internationalisation module for PHP
ii  php7.0-json                    7.0.7-1~dotdeb+8.1           amd64        JSON module for PHP
ii  php7.0-mcrypt                  7.0.7-1~dotdeb+8.1           amd64        libmcrypt module for PHP
ii  php7.0-memcached               2.2.0-2-52c644b-1~dotdeb+8.1 amd64        memcached extension module for PHP, uses libmemcached
ii  php7.0-msgpack                 2.0.1-1~dotdeb+8.1           amd64        MessagePack serializer for PHP
ii  php7.0-mysql                   7.0.7-1~dotdeb+8.1           amd64        MySQL module for PHP
ii  php7.0-opcache                 7.0.7-1~dotdeb+8.1           amd64        Zend OpCache module for PHP
ii  php7.0-readline                7.0.7-1~dotdeb+8.1           amd64        readline module for PHP
ii  php7.0-sqlite3                 7.0.7-1~dotdeb+8.1           amd64        SQLite3 module for PHP
```

You are absolutely right of course in saying that the PHP7.0 ticket is as yet unresolved, however, as far as I can tell most of the work should be completed and allowing package installations would facilitate testing somewhat – which is actually why I opened this ticket.

**Comment by Okeanos** [comment]
Just wondering whether there was any update to this. It's been about a months since I opened the issue and even a quick "not any time soon, check back in a month or two" would be appreciated at this point.

**Comment by mattab** [comment]
Hello guys,

FYI: PHP7 is now officially supported in Piwik, so it would be awesome to have the support in Debian package :+1:

**Comment by aureq** [comment]
Marking as fixed with commit  d229c338804cc691497955313d9eb3cbc2c23caa
@Okeanos you should be good to go.

**Comment by Okeanos** [comment]
Leaving issues #40 and #44 aside the installation works fine with PHP7 👍 .
